browser/scripts/ui.js

```diff
@@ -465,11 +465,22 @@ VizorUI.prototype.openPresetSaveDialog = function(serializedGraph) {
 };
 
 VizorUI.prototype.setWorldEditorMode = function(is_active) {
+	var dom = this.dom;
 	is_active = !!is_active;	// force bool
-	if (is_active)
+	if (is_active) 
 		this.viewmode = uiViewMode.world_editor;
-	else
+	else 
 		this.viewmode = uiViewMode.patch_editor;
+	
+	dom.btnSavePatch.attr('disabled',is_active);
```

Comment:
leaving this for reference here
http://stackoverflow.com/questions/13626517/remove-disabled-attribute-using-jquery